www/PSPDFKit.js

```diff
@@ -201,6 +201,28 @@ exports.saveDocument = function(success, error) {
   exec(success, error, "PSPDFKitCordovaPlugin", "saveDocument");
 };
 
+/**
+ * Imports all annotations from the specified XFDF file to the current document.
+ * 
+ * @param xfdfFile XFDF file URI to import annotations to
+ * @param success Success callback function.
+ * @param error Error callback function
+ */
+exports.importXfdf = function(xfdfFile, success, error) {
```

Comment:
Is this the same API exposed by iOS?

Reply:
Yes, the API follows iOS guide.

Reply:
Yes. Here's the iOS PR which adds XFDF import/export support https://github.com/PSPDFKit/Cordova-iOS/pull/77